docs/src/whatsnew/3.1.rst

```diff
@@ -1,6 +1,6 @@
 .. include:: ../common_links.inc
 
-v3.1 (17 Aug 2021) [unreleased]
+v3.1 (15 Sep 2021)
```

Comment:
Ok but, as documented in the release docs, we still need to [remove "[unreleased"] from the line anyway](https://github.com/SciTools/iris/blame/v3.0.0/docs/iris/src/developers_guide/release.rst#L136), and it then comes back again when we [make a new 'latest.rst' from the template](https://github.com/SciTools/iris/blame/v3.0.0/docs/iris/src/developers_guide/release.rst#L167-L170).

So if we want to adopt this, I think it needs to be in the template.

will address this ...

Reply:
There is another slight problem with this, in that the automated form presents the release version as "3.1.0", whereas in the past we have only quoted "major.minor".

Reply:
Later ...
Actually, this is a really **_bad_** idea, because if you don't replace with literal content then on the next release it will say "3.2 <current_date>" at the top of the 3.1 page, as well as the new 3.2 one.

So, I think we should continue to write these as fixed content when crafting final release notes as we have done in the past.
So :-1: for me